presto-main/src/main/java/com/facebook/presto/sql/planner/DomainTranslator.java

```diff
@@ -16,6 +16,7 @@
 import com.facebook.presto.Session;
 import com.facebook.presto.metadata.Metadata;
 import com.facebook.presto.metadata.Signature;
+import com.facebook.presto.spi.ConnectorSession;
```

Comment:
"Optimize query planning for simple IN clauses" cmt msg -- you could be more specific in the commit title that you improve `DomainTranslator`. "Optimize query planning" sounds much more general.